.gitignore

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
I guess we have a conflict that should be resolved

Reply:
ok